MerlinAU.sh

```diff
@@ -3493,6 +3518,7 @@ _high_risk_phrases_interactive_()
                 if ! _WaitForYESorNO_
                 then
                     Say "Exiting for changelog review."
+                    Update_Custom_Settings "FW_New_Update_Changelog_Approval" "BLOCKED"
```

Comment:
Good catch, I didn't think about someone being extremely eager to update the second it becomes available and didn't let the script do it's overnight check and trigger first.